example.main.scss

```diff
@@ -2,7 +2,7 @@
    INUITCSS
    ========================================================================== */
 
-/*!*
```

Comment:
I couldn’t find a reason for this style so I adapted it to the style used anywhere else.